components/organisms/ContributorProfilePage/contributor-profile-page.tsx

```diff
@@ -97,6 +97,8 @@ const ContributorProfilePage = ({
     display_local_time: displayLocalTime
   } = user || {};
 
+  const iscConnected = !!user?.is_open_sauced_member;
```

Comment:
we don't need the `!!user` conversion in the check
```suggestion
  const isConnected = user ? user.is_open_sauced_member : false ;
```

Reply:
This is the same thing. Just more characters.